libraries/gpu-gl-common/src/gpu/gl/GLBackend.cpp

```diff
@@ -105,13 +107,27 @@ GLBackend::CommandCall GLBackend::_commandCalls[Batch::NUM_COMMANDS] =
 };
 
 #define GL_GET_INTEGER(NAME) glGetIntegerv(GL_##NAME, &const_cast<GLint&>(NAME)); 
-    
+
+#define BYTES_PER_KIB 1024L
+#define BYTES_PER_MIB (1024L * BYTES_PER_KIB)
```

Comment:
I suggest using `BYTES_PER_KILOBYTE` and `BYTES_PER_MEGABYTE`. The rest of the codebase has "kilobyte" and "megabyte" spelt out, and it's not clear what a "KIB" or "MIB" is (e.g, whether these are special things).

Or use/update NumericalContstants.h.

Reply:
KiB and MiB are traditional Kilobytes and Megabytes. This is a common thing in the Linux world, as a lot of software (like windows) uses traditional and decimal interchangeably without the end user seeing that.